public/js/common.js

```diff
@@ -68,3 +68,113 @@ function failSnackbar(msg) {
 $(document).on('click', '#closeSnack', function() {
     $(this).parent().fadeOut();
 });
+
+
+/*
+These errors are only for UI use
+Error codes and their corresponding message. Error codes are under different
+categories:
+1000 -> system
+2000 -> user
+3000 -> question
+4000 -> class
+5000 -> analytics
+6000 -> import/export
+7000 -> settings
+*/
+
+const errors = Object.freeze({
+    //1000 system
+    1000: 'Invalid Request',
+    1001: 'Missing Requirement',
+    1002: 'Permission Denied',
+    1003: 'Confirm password doesn\'t match',
```

Comment:
this is a user issue not the system

Reply:
I am not sure what it falls under, technically it has nothing to do with users, and so I thought it would be appropriate to add here